ui/components-react/src/test/filter-builder/FilterBuilderState.test.ts

```diff
@@ -116,14 +116,52 @@ describe("usePropertyFilterBuilderState", () => {
     const { result } = renderHook(() => usePropertyFilterBuilderState());
     const { actions } = result.current;
 
+    let rootGroup = result.current.state.rootGroup;
+    rootGroup.items.push({ id: "testId", groupId: "groupId" });
```

Comment:
You shouldn't mutate object stored in React state. Use `actions` to setup state for tests.

Reply:
fixed